### DISCUSSION TITLE
What is the status of this project?

### DISCUSSION THREAD

**Discussion by lechu445** [jhuckaby/Cronicle#580] [General]
My obeservations: 
- most of pull requests wait 2 years
- over 200 issues are reported
- last commits are only security issues fixes
- there are only a few contributors
- [this graph](https://github.com/jhuckaby/Cronicle/graphs/contributors) shows that intense development ended in 2018 

So, my questions:
1. What is the future of this project?
2. Shall we consider this project as still maintained?
3. Are you looking for new contributors/maintainers/etc?
4. Are you looking for new funds to keep this project?

I think this answers will be beneficial for these people that decide if use Cronicle in theirs project.

@jhuckaby

**Comment by jhuckaby** [comment]
> most of pull requests wait 2 years

We don't accept pull requests.  See our [pull request template](https://github.com/jhuckaby/Cronicle/blob/master/.github/pull_request_template.md) for an explanation.  I don't have a bot that auto-closes them, so they stack up.

> over 200 issues are reported

I don't have a bot that auto-closes issues for inactivity, so they stack up.  Many issues are years old and out of date.

> last commits are only security issues fixes

Cronicle v1 is a stable product, and is in maintenance mode.  I am hard at work on Cronicle v2 (Orchestra) which is a ground-up rewrite, and will have a new GitHub repo.  It will be released in 2023.

> there are only a few contributors

Is that a bad thing?  Cronicle is just a personal side project for me, which I wrote on my own.  I am not looking for contributors at this time, and we don't merge pull requests.

> [this graph](https://github.com/jhuckaby/Cronicle/graphs/contributors) shows that intense development ended in 2018

See above.  Cronicle v1 is a stable product, and is in maintenance mode.  Intense development is happening, just not on this repo 😉 

> What is the future of this project?

Cronicle v1 will be maintained for the foreseeable future.  Cronicle v2 (Orchestra) will be released separately, and has feature parity, so it "may" replace it one day (depends on user interest).

> Shall we consider this project as still maintained?

Yes.

> Are you looking for new contributors/maintainers/etc?

Not at this time, but possibly when Cronicle v2 (Orchestra) is released.

> Are you looking for new funds to keep this project?

Nope, but thank you for asking.  Cronicle is just a personal project of mine, which ended up getting some interest from people, to my surprise and delight.  But I'm not ready to accept donations, because v1 is now in maintenance mode.  I'd feel guilty accepting any money.
